sdk/armcore/poller.go

```diff
@@ -452,6 +454,16 @@ func (pt *pollingTrackerBase) updateRawBody() error {
 			pt.Err = err
 			return pt.Err
 		}
+		// seek back to the beginning of the body or reassign the information to the body
+		if seeker, ok := pt.resp.Body.(io.Seeker); ok {
+			_, err = seeker.Seek(0, io.SeekStart)
+			if err != nil {
+				return err
```

Comment:
From the above code I believe `err` should be assigned to `pt.Err` before returning yes?

Reply:
Yes I think so. Done.